# "tts --list_models" to get a complete list of available models
# just working with xtts_v2 for now
tts:
  model_name: tts_models/multilingual/multi-dataset/xtts_v2
  config_file_name: config.json
  use_deepspeed: true
  use_cuda: true
  temperature: 0.7
